### PR TITLE
Bugfix: add check that layer has contrast slider

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
This is a fix for https://github.com/haesleinhuepf/napari-brightness-contrast/issues/7
As suggested in that issue thread, this PR adds a check that a layer has contrast sliders to adjust.
The test for this condition is in PR: https://github.com/haesleinhuepf/napari-brightness-contrast/pull/8
Hopefully that test will pass once this fix is merged. Fingers crossed that I didn't mess something up!

### CODE REVIEW

**Review by haesleinhuepf — COMMENTED** [review]
Hi Peter @psobolewskiPhD ,

these changes look to me like code-formatting changes only. Can you point me to the actual bugfix? Alternatively, feel free to send a PR with the bugfix only.

Thanks and sorry for my confusion.

Best,
Robert

**Comment by psobolewskiPhD** [comment]
Yeah, sorry I mentioned this in the issue: i had black in my env and it auto formatted.
The fix is line 209:            ` if hasattr(layer.events, "contrast_limits"):`
in accordance with your suggestion:
https://github.com/haesleinhuepf/napari-brightness-contrast/issues/7#issuecomment-1013707427

File: napari_brightness_contrast/_dock_widget.py [hunk 1]
```diff
@@ -177,25 +197,28 @@ def redraw(self, rebuild_gui=True):
...
 
         # patch events
         selected_layers = self.selected_image_layers()
         for layer in self.viewer.layers:
-            layer.events.contrast_limits.disconnect(self._data_changed_event)
-            layer.events.data.disconnect(self._data_changed_event)
-            if layer in selected_layers:
-                layer.events.contrast_limits.connect(self._data_changed_event)
-                layer.events.data.connect(self._data_changed_event)
+            if hasattr(layer.events, "contrast_limits"):
```
Comment (psobolewskiPhD) [inline napari_brightness_contrast/_dock_widget.py:209]: Here, this is the key line you suggested, to check that "contrast limits" can be changed.

Comment (haesleinhuepf) [reply napari_brightness_contrast/_dock_widget.py:209]: ok cool, thanks!

**Comment by haesleinhuepf** [comment]
Thanks Peter @psobolewskiPhD !

**Comment by psobolewskiPhD** [comment]
BTW, I see tests are passing; would hope that happens after the other PR for tests!
fingers crossed

**Pull request merged**